guides/source/layout.html.erb

```diff
@@ -39,59 +45,83 @@
         <li class="more-info"><a href="https://github.com/rails/rails">Contribute on GitHub</a></li>
       </ul>
     </div>
-  </div>
-  <div id="header">
+  </nav>
+  <header id="page_header">
     <div class="wrapper clearfix">
-      <div class="header-logo"><a href="index.html" title="Return to home page">Guides.rubyonrails.org</a></div>
-      <ul class="nav">
-        <li><a class="nav-item" href="index.html">Home</a></li>
-        <li class="guides-index guides-index-large">
-          <a href="index.html" id="guidesMenu" class="guides-index-item nav-item">Guides Index</a>
-          <div id="guides" class="clearfix" style="display: none;">
-            <hr />
-            <dl class="guides-section-container">
-              <% documents_by_section.each do |section| %>
-                <div class="guides-section">
-                  <dt><%= section['name'] %></dt>
+      <nav id="feature_nav"> <%# we use nav as an element to also provide the aria role and improve semantic meaning. %>
+        <div class="header-logo">
+          <a href="index.html" title="Return to home page">Guides</a>
```

Comment:
This is [already] a bad title: "home page" sounds more like rubyonrails.org than the guides index/list/catalog/home. Perhaps we should fix it while we're here?

Reply:
Improved to be explicit in 7e40297a